gpflow/utilities/ops.py

```diff
@@ -6,6 +6,9 @@
 import numpy as np
 
 
+EllipsisType = type(...)
```

Comment:
mypy won't like this. perhaps that's not a problem esp since GPflow doesn't use mypy yet. The error is

> sandbox.py:4: error: Variable "sandbox.EllipsisType" is not valid as a type

Reply:
I don't however have another suggestion beyond `Any`